composer.json

```diff
@@ -186,12 +185,14 @@
     },
     "scripts": {
         "post-install-cmd": [
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
+            "cp -n project-base/config/parameters.yaml.dist project-base/config/parameters.yaml",
+            "cp -n project-base/config/parameters_test.yaml.dist project-base/config/parameters_test.yaml",
```

Comment:
`cp -n ` returns 1 on mac. It's impossible for me to install application if parameters.yaml (parameters_test.yaml) exists. 

it could work with 
```
cp -n project-base/config/parameters.yaml.dist project-base/config/parameters.yaml || true
```
or 
```
cp -n project-base/config/parameters.yaml.dist project-base/config/parameters.yaml || echo '[skiped] parameters.yaml exists'
```

Reply:
fixed